tests/test_chebyMap.py

```diff
@@ -0,0 +1,493 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import numpy as np
+import numpy.testing as npt
+
+import astshim
+from astshim.test import MappingTestCase
+
+
+def cheby(n, x):
+    """Reference implementation of Chebyshev polynomials of the first kind
+
+    f(x) = T_n(x)
```

Comment:
Yes. Sorry for the lousy documentation. I've improved it. But as you note below, numpy provides its own Chebyshev implementation so perhaps I should just use that. I'll see how much more work that would be (since this code is already written and tested).